go/protocol/keybase1/extras.go

```diff
@@ -1351,6 +1351,11 @@ func (u UserPlusKeysV2AllIncarnations) FindKID(kid KID) (*UserPlusKeysV2, *Publi
 	return nil, nil
 }
 
+func (u UserPlusKeysV2AllIncarnations) HasKID(kid KID) bool {
```

Comment:
I'd be a little surprised that HasKID includes previous incarnations. Add comment?